src/copyediting/models.py

```diff
@@ -39,7 +39,7 @@ class CopyeditAssignment(models.Model):
     date_decided = models.DateTimeField(null=True, blank=True)
 
     files_for_copyediting = models.ManyToManyField('core.File', related_name='files_for_copyediting')
-    copyeditor_files = models.ManyToManyField('core.File')
+    copyeditor_files = models.ManyToManyField('core.File', blank=True)
```

Comment:
Just a Q. Does the admin it let you save the form if it the field is not nullable but you post a blank field?

Reply:
Yeah, its stored in a secondary table so doesn't need to be nulled but the without blank the form forces you to select one.

Reply:
weird, I didn't know this quirk about M2M fields